app/models/katello/authorization/pool.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module Authorization::Pool
+    extend ActiveSupport::Concern
+
+    include Authorizable
+
+    def readable?
+      authorized?(:view_pools)
```

Comment:
Currently, we expose subscription permissions to the user which makes sense given that is what we present to the user. This would introduce having to manage pool and subscription permissions. I'd consider switching to using the subscription permissions to reduce overhead/headache for users.